src/volume_calc.cpp

```diff
@@ -225,6 +225,13 @@ vector<VolumeCalculation::Result> VolumeCalculation::execute() const
     iterations++;
     size_t total_samples = iterations * n_samples_;
 
+    // warn user if total sample size is greater than what the size_t type can
+    // represent
+    if (total_samples > SIZE_T_MAX) {
+      warning("The number of samples has exceeded the size_t type. Volume "
+              "results may be inaccurate.");
+    }
+
```

Comment:
I'd propose getting rid of this because
1. By definition, a variable of `size_t` _can't_ exceed `SIZE_T_MAX`, so this conditional will never get hit, and
2. Even if it were possible, from a practical standpoint it would never happen. You could do 1e9 iterations with 1e9 samples and still not hit it.
```suggestion
```

Reply:
Meant that to be `>=` 🤦🏻, but you're right that since `total_samples` can't be `>` it could just be `==`.

As for the second point, I agree that it would be an outrageous volume calculation to hit that warning. My thought was that if someone _did_ run that many samples they would have no way of knowing that the volumes would be off because `total_samples` would stop increasing, so I figured it wouldn't hurt to drop it in there.